src/app/utils.js

```diff
@@ -1,20 +1,20 @@
 export function getReadableFileSizeString(fileSize) {
-    let fileSizeInBytes = parseInt(fileSize);
-    let i = -1;
-    const byteUnits = [' kB', ' MB', ' GB', ' TB', 'PB', 'EB', 'ZB', 'YB'];
-    do {
-        fileSizeInBytes = fileSizeInBytes / 1024;
-        i++;
-    } while (fileSizeInBytes > 1024);
+  let fileSizeInBytes = parseInt(fileSize);
+  let i = -1;
+  const byteUnits = [` kB`, ` MB`, ` GB`, ` TB`, `PB`, `EB`, `ZB`, `YB`];
```

Comment:
I love that we included stuff all the way up to yottabyte. You can never be too sure with gifs.